tests/config/config.drone.json

```diff
@@ -36,5 +36,7 @@
       "JbdgMGkt3Cq6B0u"
     ],
     "testContent": "123456",
-    "expirationDate": "2022-10-01"
+  "expirationDate": "2022-10-01",
+  "deletedFolderId": "2147596415",
+  "deletedFileId": "2147596419"
```

Comment:
if those values are only used in `tests/fileTrashTest.js` then lets put them there